components/dirigeants-section/summary.tsx

```diff
@@ -53,8 +57,10 @@ const DirigeantSummary: React.FC<IDirigeants> = ({
   if (inseeDirigeant) {
     summaries.push(<a href="#insee-dirigeant">{inseeDirigeant}</a>);
   }
-
-  if (!isAPINotResponding(immatriculationRNE)) {
+  if (
+    !isAPILoading(immatriculationRNE) &&
+    !isAPINotResponding(immatriculationRNE)
+  ) {
```

Comment:
Would be nice to work on `<DataSummary />` one day as it is a very similar logic to `<DataSection />`

Maybe in #667